receiver/udplogreceiver/README.md

```diff
@@ -69,6 +70,16 @@ Other less common encodings are supported on a best-effort basis.
 See [https://www.iana.org/assignments/character-sets/character-sets.xhtml](https://www.iana.org/assignments/character-sets/character-sets.xhtml)
 for other encodings available.
 
+#### `async` configuration
+
+If set, the `async` configuration block instructs the `udp_input` operator to read and process logs asynchronsouly and concurrently.
+
+**note** If `async` is not set at all, a single thread will receive lines synchronously.
+
+| Field                                   | Default              | Description |
+| ---                                     | ---                  | ---         |
+| `fixed_reader_routine_count`            | 1                    | Concurrency level - Determines how many go routines read from UDP port (and process logs before sending downstream). |
```

Comment:
I think a more concise name can be sufficiently clear. WDYT of just `readers`, given that it is within an `async` block and will have a data type and documentation which indicate that it's a count?